tests/config/ios/INTERFACE_DOWN/time_synchronized/yang.json

```diff
@@ -0,0 +1,33 @@
+{
+	"yang_message": {
+		"interfaces": {
+			"interface": {
+				"GigabitEthernet2": {
+					"state": {
+						"admin_status": "DOWN"
+					}
+				}
+			}
+		}
+	},
+	"message_details": {
+		"facility": 23,
+		"time": "08:30:56",
+		"pri": "189",
+		"host": "router1",
```

Comment:
These details need to match the syslog message. If you add the exact message that I provided the tests should pass.